ttnn-visualizer/src/components/GoldenTensorComparisonIndicator/index.tsx

```diff
@@ -0,0 +1,38 @@
+import './styles.scss';
+
+const GOLDEN_GLOBAL = [1, 0.9999, 0.9888, 1, 1, 0.8888];
+const GOLDEN_LOCAL = [1, 0.9999, 1, 0.9888, 1, 0.99];
+
+type CalculationOptions = 'global' | 'local';
+
+interface GoldenTensorComparisonIndicatorProps {
+    index: number;
+    calc: CalculationOptions;
+}
+
+function GoldenTensorComparisonIndicator({ index, calc }: GoldenTensorComparisonIndicatorProps) {
+    const source = calc === 'local' ? GOLDEN_LOCAL : GOLDEN_GLOBAL;
+
+    return (
+        <>
+            <div
+                className='performance-square'
+                style={{
+                    backgroundColor: calculateOpPerformanceColor(source[index]),
+                }}
+            />
+            <span>t{source[index].toFixed(4)}</span>
+        </>
+    );
+}
+
+const calculateOpPerformanceColor = (value: number): string => {
```

Comment:
this probably belongs in utils/ of sort as we would likely be using that color functionality other places

Reply:
Moved to a functions folder.